server/routes/lecture-route.js

```diff
@@ -10,5 +10,6 @@ router.get("/bookable", lectureRoutes.getBookingLectures);
 router.post("/:lectureId/book", lectureRoutes.newBooking);
 router.get("/previousbooking", lectureRoutes.getExistentBooking);
 router.get("/:lectureid/students", lectureRoutes.getBookedStudents);
+router.post("/:lectureid/cancelbook", lectureRoutes.cancelBooking);
```

Comment:
Since this API removes a booking from the database that was previously inserted by POST /:lectureId/book, I would suggest to use the DELETE method on the same route: DELETE /:lectureId/book

Reply:
You're right! I'll change asap

Reply:
I don't know if I have problem with this link or because I am working on branch PULSEBS-7 It shows me 400 http error, I used DELETE /lectures/:lectureid